phpstan.neon.dist

```diff
@@ -0,0 +1,16 @@
+parameters:
+  level: 5
```

Comment:
This number goes up to 9. This makes the code very strict and harder to maintain, we can up this number at a later time.